src/pages/projects.jsx

```diff
@@ -59,6 +60,48 @@ const ProjectImage = styled.img`
     max-width: 100%;
 `;
 
+const Player1 = ({ classCircles }) => (
+    <ReactPlayer
+        url={Circles}
+        className={classCircles}
+        width="100%"
+        height="100%"
+        controls={false}
+        muted
+        playing
+        loop
+    />
+);
+
+const CirclesVideo = styled(Player1)`
+    margin: 0 auto 100px auto;
+    display: block;
+    width: auto;
+    height: auto;
+    max-width: 100%;
+`;
+
+const Player2 = ({ classCityscape }) => (
+    <ReactPlayer
+        url={Cityscape}
+        className={classCityscape}
+        width="100%"
+        height="100%"
+        controls={false}
+        muted
+        playing
+        loop
+    />
+);
+
+const CityscapeVideo = styled(Player2)`

```

Comment:
`CirclesVideo` and `CityscapeVideo` are exactly the same, except one takes in `Player1` and the other takes in `Player2`. Since both are the same, you should just make one styled component off of `div`, which can hold any video player in it. This way you only need one video player styled component.

Reply:
Gotcha, thank you very much!